docs/reference/glossary.asciidoc

```diff
@@ -24,28 +24,46 @@ Also see <<glossary-text,text>> and <<glossary-term,term>>.
 // end::analysis-def[]
 --
 
+[[glossary-auto-follow-pattern]] auto-follow pattern ::
+// tag::auto-follow-pattern-def[]
+An index pattern pattern used to automatically configure new indices as 
```

Comment:
dedupe `pattern`
```suggestion
An index pattern used to automatically configure new indices as 
```
I think a link to either a definition or docs for index pattern would be helpful here. Because we plan to reuse the content, this would need to formatted as an external link.